modules/installation-creating-gcp-vpc.adoc

```diff
@@ -2,6 +2,11 @@
 //
 // * installing/installing_gcp/installing-gcp-user-infra.adoc
 // * installing/installing_gcp/installing-restricted-networks-gcp.adoc
+// * installing/installing_gcp/installing-gcp-user-infra-vpc.adoc
+
+ifeval::["{context}" == "installing-gcp-user-infra-vpc"]
+:shared-vpc:
+endif::[]
 
 [id="installation-creating-gcp-vpc_{context}"]
 = Creating a VPC in GCP
```

Comment:
We use the network when generating ignition files. So we need to move it ahead of that section. I suggest to have it as a 2-level section under Configuring the GCP project that hosts your shared VPC network. When moving the section, please keep an eye on the variables in 01_vpc.yaml to make sure they're defined before or we can use <> to describe them directly.